data/events.json

```diff
@@ -1,4 +1,16 @@
 [{
+<<<<<<< HEAD
+    "title": "No upcoming events",
```

Comment:
Please refer #132 and work towards making a poster and keeping Thursday meetups as a passive event.

Reply:
Which type of poster it should be. I mean what should it consist. Which type of details we can keep in it.

Reply:
Just a normal poster, regarding Thursday meetups. 
Venue - Ask on the group 
Time - Ask on the group
OD - No
Registration links - N/A
Open For All

Description - An open for all weekly meetup to discuss, ideas, projects and future events with members.